force-app/main/default/classes/Homework.cls

```diff
@@ -90,6 +103,20 @@ public with sharing class Homework
         // set result to 0 if first and second are equal
         // set result to 1 if first is greater than second 
         // set result to -1 if first is less than second 
+        if (first == second)
+        {
+            result = 0;
+        }
+
+        if (first > second)
+        {
+            result = 1;
+        }
+
+        if (first < second)
+        {
+            result = -1;
+        }
```

Comment:
This is slightly less efficient than it could be, as you will execute all of the if statements every time. If first is equal to second, none of the others can be true so you can avoid them using an else statement. You can also default to the -1 result rather than check the third.  E.g.

```
Integer result=-1;
if (first==second) 
{
    result=0;
}
else if (first > second) 
{
    result=1;
}
```